pkg/pipeline/engine/jenkins/mappper.go

```
@@ -293,25 +293,46 @@ func injectResources(container *v1.Container, cpuLimit string, cpuRequest string
 		if container.Resources.Limits == nil {
 			container.Resources.Limits = v1.ResourceList{}
 		}
-		container.Resources.Limits[v1.ResourceCPU] = resource.MustParse(cpuLimit)
+		quantity, err := resource.ParseQuantity(cpuLimit)
+		if err != nil {
+			logrus.Warnf("invalid cpuLimit %q, ignoring the config", cpuLimit)
```

do we need to return the error?

---

Updated to return the error